### PR TITLE
V0.1 release

### PR DESCRIPTION
**Pull request by blootsvoets**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+MONGODB_DIR=/usr/local/var/lib/docker/mongodb
+HOTSTORAGE_USERNAME=<mongodb-user>
+HOTSTORAGE_PASSWORD=<mongodb-password>
+HOTSTORAGE_NAME=<mongodb-database>
+IS_VERSION=5.3.0
```
Comment (nivemaham) [inline .env]: I think we can remove `IS_VERSION` here. Are we using hotstorage env variables to this deployment?

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+```
+
+Now the gateway is accessible through <http://localhost:8080/radar-gateway/>.
+
+Now you can access the REST proxy via the gateway:
+```shell
+curl -H 'Authorization: Bearer myAccessToken' http://localhost:8080/radar-gateway/topics
+```
+
+The access token should be generated by the aforementioned Management portal.  The gateway does content validation for posted data. It requires to use the Avro format with JSON serialization, using the `application/vnd.kafka.avro.v1+json` or `application/vnd.kafka.avro.v2+json` media types, as described in the [REST Proxy documentation].  It also requires messages to have both a key and a value, and if the key contains a user ID, it needs to match the user ID that is used in the for authentication.
```
Comment (nivemaham) [inline README.md]: Should we add some description about `UserID` and `SourceID` filtering?

File: src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <servlet-class>org.mitre.dsmiley.httpproxy.ProxyServlet</servlet-class>
     <init-param>
       <param-name>targetUri</param-name>
-      <param-value>http://localhost:8082/</param-value>
+      <param-value>http://localhost:8082</param-value>
```
Comment (nivemaham) [inline src/main/webapp/WEB-INF/web.xml:12]: How can we change this to use different url ? Would that be necessary to integrate this with docker-compose stack?

**Review by nivemaham — APPROVED** [review]
Looks very neat and nice. Very interesting to see the use of Kotlin? Was there any particular reason to do the validation using Kotlin? I left few minor comments that came into my mind.

File: README.md (hunk 2 above)
Comment (blootsvoets) [reply README.md]: Yes, good point

File: .env (hunk 1 above)
Comment (blootsvoets) [reply .env]: Good point, I'll remove them

File: src/main/webapp/WEB-INF/web.xml (hunk 3 above)
Comment (blootsvoets) [reply src/main/webapp/WEB-INF/web.xml:12]: For this there is a separate file: `src/main/docker/web.xml`. The current file can be used for local testing.

Comment (nivemaham) [reply src/main/webapp/WEB-INF/web.xml:12]: Right. I noticed the two files. But missed to notice the tiny detail. Nice

**Pull request merged**